src/applications/edu-benefits/1990e/manifest.json

```diff
@@ -2,5 +2,6 @@
   "appName": "22-1990E Education benefits form",
   "entryFile": "./edu-benefits-entry.jsx",
   "entryName": "1990e-edu-benefits",
-  "rootUrl": "/education/apply-for-education-benefits/application/1990E"
+  "rootUrl": "/education/apply-for-education-benefits/application/1990E",
+  "e2eTestsDisabled": "true"
```

Comment:
@bradl3yC By adding this key, it would disable the tests from running. Could the feature toggle be used in the e2e test, erasing the need for this?

Reply:
this should be removed, should it not?